smac/utils/io/cmd_reader.py

```diff
@@ -84,96 +273,456 @@ class CMDReader(object):
+                               default='.', type=str,
+                               help="[dev] Specifies the path to the execution-directory.")
+        scen_opts.add_argument("--deterministic", dest="deterministic",
+                               default=False, type=truthy,
+                               help="[dev] If true, the optimization process will be "
+                                    "repeatable.")
+        scen_opts.add_argument("--run-obj", "--run_obj", dest="run_obj",
+                               type=str, action=ProcessRunObjectiveAction,
+                               required=True, choices=['runtime', 'quality'],
+                               help="[dev] Defines what metric to optimize. When "
+                                    "optimizing runtime, *cutoff_time* is "
+                                    "required as well.")
+        self.overall_obj_arg = \
+            scen_opts.add_argument("--overall-obj", "--overall_obj", dest="overall_obj",
+                                   type=str, action=ParseOverallObjectiveAction, default='par10',
+                                   help="[dev] PARX, where X is an integer defining the "
+                                        "penalty imposed on timeouts (i.e. runtimes that "
+                                        "exceed the *cutoff-time*).")
+        scen_opts.add_argument("--cost-for-crash", "--cost_for_crash", dest="cost_for_crash",
+                               default=float(MAXINT), type=float,
+                               help="[dev] Defines the cost-value for crashed runs "
+                                    "on scenarios with quality as run-obj.")
+        scen_opts.add_argument("--cutoff-time", "--cutoff_time", dest="cutoff",
+                               default=None, type=float,
+                               help="[dev] Maximum runtime, after which the "
+                                    "target algorithm is cancelled. **Required "
+                                    "if *run_obj* is runtime.**")
+        scen_opts.add_argument("--memory-limit", "--memory_limit", dest="memory_limit",
+                               type=float,
+                               help="[dev] Maximum available memory the target algorithm "
+                                    "can occupy before being cancelled in MB.")
+        scen_opts.add_argument("--tuner-timeout", "--tuner_timeout", dest="algo_runs_timelimit",
+                               default=float('inf'), type=float,
+                               help="[dev] Maximum amount of CPU-time used for optimization.")
+        scen_opts.add_argument("--wallclock-limit", "--wallclock_limit", dest="wallclock_limit",
+                               default=float('inf'), type=float,
+                               help="[dev] Maximum amount of wallclock-time used for optimization.")
+        scen_opts.add_argument("--always-race-default", "--always_race_default", dest="always_race_default",
```

Comment:
That's not a scenario option, but a dev option

Reply:
Changed that as discussed in the meeting.